lib/index.js

```diff
@@ -4,5 +4,14 @@ import bootstrapUI from 'ui';
 // bootstrap UI
 bootstrapUI();
 
+// expose on window
+if (window) {
```

Comment:
Please remove this, this is not needed.

Bdsm is already exposed on the window via webpack when appropriate.